src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/CollectionPropertiesShouldBeReadOnlyTests.cs

```diff
@@ -194,6 +207,48 @@ class A
 ");
         }
 
+        [Fact]
+        public void CSharp_CA2227_NonCollection()
+        {
+            VerifyCSharp(@"
+using System;
+
+public class A
+{
+    public string Name { get; set; }
+}
+");
+        }
+
+        [Fact, WorkItem(1900, "https://github.com/dotnet/roslyn-analyzers/issues/1900")]
+        public void CSharp_CA2227_ReadOnlyCollection()
```

Comment:
💡 This test name is misleading since it checks for `IReadOnlyCollection<T>` and not `ReadOnlyCollection<T>`.

```suggestion
        public void CSharp_CA2227_IReadOnlyCollection()
```